lib/widgets/scrollbar.dart

```diff
@@ -116,61 +142,36 @@ class _AppScrollbarState extends State<AppScrollbar> {
       child: MediaQuery.removePadding(
         context: context,
         removeTop: true,
-        child: NotificationListener<ScrollNotification>(
-          onNotification: (notification) {
-            final Offset? dragPosition;
-            if (notification is ScrollStartNotification) {
-              dragPosition = notification.dragDetails?.localPosition;
-            } else if (notification is ScrollUpdateNotification) {
-              dragPosition = notification.dragDetails?.localPosition;
-            } else if (notification is OverscrollNotification) {
-              dragPosition = notification.dragDetails?.localPosition;
-            } else {
-              dragPosition = null;
-            }
-            final windowSize = context.size;
-            setState(() {
-              showScrollLabels = windowSize != null &&
-                  dragPosition != null &&
-                  (Directionality.of(context) == TextDirection.rtl
-                      ? (dragPosition.dx < scrollLabelDragAreaWidth && dragPosition.dx >= 0)
-                      : (dragPosition.dx > windowSize.width - scrollLabelDragAreaWidth &&
-                          dragPosition.dx <= windowSize.width));
-            });
-            return false;
-          },
-          child: Stack(
-            children: [
-              Scrollbar(
-                controller: controller,
-                thumbVisibility: widget.thumbVisibility,
-                trackVisibility: widget.showTrackOnHover,
-                thickness: widget.thickness,
-                radius: widget.radius,
-                notificationPredicate: widget.notificationPredicate,
-                interactive: widget.interactive,
-                child: Theme(
-                  data: theme.copyWith(highlightColor: highlightColor),
-                  child: widget.child,
-                ),
+        child: widget.labelBuilder == null
+            ? scrollbarChild
+            : NotificationListener<ScrollNotification>(
+                onNotification: (notification) {
```

Comment:
I'd prefer if the tree didn't change when this property changes, since this means that the children State's will be lost.

We can just return false from the notification listener.

Reply:
Ah, yeah, that's better 👍